pkg/generators/extension.go

```diff
@@ -71,6 +71,10 @@ func (e extension) allowedValues() sets.String {
 	return tagToExtension[e.idlTag].allowedValues
 }
 
+func (e extension) kind() types.Kind {
```

Comment:
This is talking about the kind of go type the extension applies to? The name is a bit confusing but I don't have a better suggestion. Maybe change in a follow-up if you think of something better.